pkg/replica/client/client.go

```diff
@@ -407,6 +407,23 @@ func (c *ReplicaClient) SendFile(from, host string, port int32) error {
 	return nil
 }
 
+func (c *ReplicaClient) ExportVolume(snapshotName, host string, port int32, exportBackingImageIfExist bool) error {
+	syncAgentServiceClient := c.getSyncServiceClient()
```

Comment:
Just leaving a note here:
I will add an error return to getSyncServiceClient() since I need to lazy init (optimization) the client due to the way the longhorn-manager uses the engine-binary for monitoring at the moment.

Reply:
Once this PR https://github.com/longhorn/longhorn-engine/pull/646 is merged, getSyncServiceClient() will return (client, err) tuple.